src/mcts/search.cc

```diff
@@ -47,6 +47,7 @@ const char* Search::kMaxPrefetchBatchStr = "Max prefetch nodes, per NN call";
 const char* Search::kCpuctStr = "Cpuct MCTS option";
 const char* Search::kTemperatureStr = "Initial temperature";
 const char* Search::kTempDecayMovesStr = "Moves with temperature decay";
+const char* Search::kTemperatureVisitOffsetStr = "Delta to be applied to all visits before calculating temperature";
```

Comment:
That UCI parameter will overflow any chess GUI. :)
We'll separate help from UCI params eventually, but for now something shorter would be nicer.

Reply:
Shorter it is.